src/main/java/com/cburch/logisim/std/ttl/Ttl7487.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by the Logisim-evolution developers
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.std.ttl;
+
+import com.cburch.logisim.data.Value;
+import com.cburch.logisim.instance.InstancePainter;
+import com.cburch.logisim.instance.InstanceState;
+
+public class Ttl7487 extends AbstractTtlGate {

```

Comment:
Please add a documentation comment to the class describing its purpose and referencing the TTL part's data sheet.